### ISSUE TITLE
Impossible to compile HEAD

### ISSUE THREAD

**Issue by vbmithr** [janestreet/bin_prot#13]
```
File "src/type_class.ml", line 67, characters 196-216:
Error: Unbound value Shape.bin_shape_unit
```

**Comment by vbmithr** [comment]
This happens only when using janestreet's opam-repository up to date. I manage to compile bin_prot with Core 114.02+05

**Comment by diml** [comment]
Indeed, I pushed too early last week. This is fixed by 48f6ce4abad2539cd384218bad35a2e8e09fdbd7
